.teamcity/_self/builds/WPComPlugins_EditorToolkit.kt

```diff
@@ -102,36 +96,51 @@ object WPComPlugins_EditorToolKit : BuildType({
 				cd apps/editing-toolkit
 				yarn build
 
-				echo
-				prev_release_build_num=${'$'}(grep build_number ../../etk-release-build/build_meta.txt | sed -e "s/build_number=//")
```

Comment:
This info is already available via the artifact output at the beginning of the job in teamcity